context/app/markdown/docs/assay/seqfish.md

```diff
@@ -98,10 +98,10 @@ In addition, all HuBMAP seq data will have an associated json file which may con
 |  "useDiagnosticMode"| Was diagnostic mode used| true|
 |  "multipointMode"| Was multipoint microscope focus mode used| false|
 |  "HandEstain"| Was H and E stain mode used| false|
-|  "channelNames"| The microscope-defined names of the fluorophore| channelNamesArray[ ]|
-|  "channelNamesArray"| List of stains, See right.| [ "635", "561," "488," "405" ]|
+|  "channelNames"| The microscope-defined names of the fluorophore| channelNamesArray |
+|  "channelNamesArray"| List of stains, See right.|  "635", "561," "488," "405" |
 |  "exposureTimes"| Length of time (usually in milliseconds) the tile is exposed to excitation light.| exposureTimesArray [ ]|
-|  "exposureTimesArray"| Exposure time for each channel within each cycle| [ [ "Cycle", "CH1", "CH2", "CH3", "CH4" ], [ "1", "1000", "1000", "1000", "700" ], ["2", "1000", "1000", "1000", "700" ], [ "3", "1000", "1000", "1000", "700"] ]|
+|  "exposureTimesArray"| Exposure time for each channel within each cycle|   "Cycle", "CH1", "CH2", "CH3", "CH4" ,  "1", "1000", "1000", "1000", "700" , "2", "1000", "1000", "1000", "700" ,  "3", "1000", "1000", "1000", "700" |
```

Comment:
No: Presumably the list-of-list structure matter: Escape the `[]` rather than deleting.